core-bundle/src/Resources/contao/widgets/SerpPreview.php

```diff
@@ -66,6 +73,15 @@ public function generate()
 		$aliasField = $this->getAliasField($suffix);
 		$descriptionField = $this->getDescriptionField($suffix);
 		$descriptionFallbackField = $this->getDescriptionFallbackField($suffix);
+		$titleTag = $model->getRelated('layout')->titleTag;
+
+		// apply title tag from layout if available
+		if ('' != $titleTag)
+		{
```

Comment:
Maybe we should move `global $objPage` here? Because we only use it inside the if condition.

Reply:
Fixed in 915066bdd.